integration/test-network/README.md

```diff
@@ -109,21 +110,22 @@ In order to initiate the FPC Chaincode enclave and register it with the FPC Encl
 
 ```bash
 cd ${FPC_PATH}/client_sdk/go/sample
-CC_ID=echo go run . -withLifecycleInitEnclave
+CC_ID=echo ORG_NAME=Org1 go run . -withLifecycleInitEnclave
 ```
+Note that we execute the go app as `Org1` and thereby issue to create and registery the FPC Chaincode enclave at `peer0.org1.example.com`.
 
-Afterwards you can run the application without the flag
+Afterwards you can run the application without the `withLifecycleInitEnclave` flag and play with multiple organizations.
```

Comment:
Actually, if by chance somebody runs "with" initEnclave, it will fail. So, it is mandatory to use it once and only once.
```suggestion
Afterwards you _must_ run the application without the `withLifecycleInitEnclave` flag and you can play with multiple organizations.
```